### PR TITLE
Remove max file count

### PR DESCRIPTION
**Pull request by kirill-sizov**
### Checklist

- [ ] I submit my changes into the `develop` branch
- [ ] I have added a description of my changes into the [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
- [ ] I have updated the documentation accordingly
- [ ] I have added tests to cover my changes
- [ ] I have linked related issues (see [GitHub docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))
- [ ] I have increased versions of npm packages if it is necessary
  ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
  [cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning),
  [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and
  [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [ ] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.

### CODE REVIEW

File: cvat/apps/engine/serializers.py [hunk 1]
```diff
@@ -836,15 +835,9 @@ def validate(self, attrs):
...
-        if (
-            (server_files := attrs.get('server_files'))
-            and attrs.get('cloud_storage_id')
-            and sum(1 for f in server_files if not f['file'].endswith('.jsonl')) > settings.CLOUD_STORAGE_MAX_FILES_COUNT
-        ):
-            raise serializers.ValidationError(f'The maximum number of the cloud storage attached files is {settings.CLOUD_STORAGE_MAX_FILES_COUNT}')
-
         filename_pattern = attrs.get('filename_pattern')
         server_files_exclude = attrs.get('server_files_exclude')
+        server_files = attrs.get('server_files')
```
Comment (Marishka17) [inline cvat/apps/engine/serializers.py:840]:
```suggestion
        server_files = attrs.get('server_files', [])
```

File: cvat/settings/base.py [hunk 2]
```diff
@@ -514,9 +514,7 @@ class CVAT_QUEUES(Enum):
 DATA_UPLOAD_MAX_MEMORY_SIZE = 100 * 1024 * 1024  # 100 MB
 DATA_UPLOAD_MAX_NUMBER_FIELDS = None   # this django check disabled
 DATA_UPLOAD_MAX_NUMBER_FILES = None
-LOCAL_LOAD_MAX_FILES_COUNT = 500
 LOCAL_LOAD_MAX_FILES_SIZE = 512 * 1024 * 1024  # 512 MB
```
Comment (Marishka17) [inline cvat/settings/base.py:517]: What about `LOCAL_LOAD_MAX_FILES_SIZE` variable? It doesn't seem to be used anywhere. Should we delete it too?

Comment (kirill-sizov) [reply cvat/settings/base.py]: Removed

**Review by Marishka17 — APPROVED** [review]

**Comment by Commifreak** [comment]
Excuse me if Iam wrong, maybe someone can point me in the right direction then, but:

I see, that `LOCAL_LOAD_MAX_FILES_SIZE` was being removed. Any I dont find it in my `base.py` as well, but: We are facing this error: `tus: unexpected response while creating upload, originated from request (method: POST, url: http://cvat.intern/api/projects/59/dataset/, response code: 413, response text: File size exceeds max limit of 26843545600 bytes, request id: n/a).`

I grepped this byte number and I found the limit inside `TUS_MAX_FILE_SIZE`. However, I updated this limit to `107374182400` and docker-compose'd down and up'ed everything, but the error message remains the same (with SAME byte amount).

Anything I forgot?

CVAT: 2.21.2

**Pull request merged**